### PR TITLE
Enhancement (ci): Improve readability of tagging for `PIPELINE=update`

### PR DESCRIPTION
**Pull request by leojonathanoh**

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -230,7 +230,7 @@ if [ "$PIPELINE" = 'build' ]; then
     fi
     date
 elif [ "$PIPELINE" = 'update' ]; then
-    GAME_IMAGE="$GAME_IMAGE_LATEST"
+    GAME_IMAGE="$GAME_IMAGE_LAYERED"
     date
     if [ ! "$NO_PULL" = 'true' ]; then
         time docker pull "$GAME_IMAGE"
```
Comment (joeltimothyoh) [inline build.sh:236]: `GAME_IMAGE="$GAME_IMAGE_LAYERED"` will not work because the image is pulled

Comment (leojonathanoh) [reply build.sh:236]: change to `docker pull "$GAME_IMAGE_LATEST"`?

File: build.sh [hunk 2]
```diff
@@ -287,7 +287,7 @@ if [ ! "$NO_PUSH" = 'true' ]; then
             time docker push "$GAME_IMAGE_LATEST"
         fi
     elif [ "$PIPELINE" = 'update' ]; then
-        time docker push "$GAME_IMAGE_LAYERED"
+        time docker push "$GAME_IMAGE_LATEST"
```
Comment (joeltimothyoh) [inline build.sh:290]: #L287 already `docker push "$GAME_IMAGE_LATEST"`

**Review by joeltimothyoh — CHANGES_REQUESTED** [review]

File: build.sh (hunk 2 above)
Comment (leojonathanoh) [reply build.sh:290]: that is for `PIPELINE=build`

File: build.sh (hunk 1 above)
Comment (joeltimothyoh) [reply build.sh]: yes

**Review by joeltimothyoh — APPROVED** [review]

**Pull request merged**